### ISSUE TITLE
Bug in the applet menu

### ISSUE THREAD

**Issue by MInessota** [linuxmint/cinnamon-spices-applets#3956]
Hi,

There is a bug in the applet menu, two repeated options appear!

![Captura-de-tela-de-2021-09-28-11-46-33](https://user-images.githubusercontent.com/28904791/135613661-8a0fdf00-3963-467c-b0ac-eb6919477368.png)

**Comment by icarter09** [comment]
What version of the applet are you using? Can you also fill out the rest of the issue template? Thanks.

**Comment by fredcw** [comment]
I don't think this is a bug in the applet because looking at the code, it only seems to show what the system reports.

`let camera_devices = this.udev_client.query_by_subsystem ("video4linux");`

It seems udev is reporting two devices. Maybe two different resolutions or something?
